Dockerfile

```diff
@@ -27,12 +27,14 @@ RUN useradd -U -m superset && \
         unzip \
         python3-pil \
         python-dev-is-python3 && \
-    curl -s https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /tmp/google-chrome-stable_current.deb && \
-    apt-get install -y --no-install-recommends /tmp/google-chrome-stable_current.deb && \
-    export CHROMEDRIVER_VERSION=$(curl -s https://chromedriver.storage.googleapis.com/LATEST_RELEASE) && \
-    curl -s https://chromedriver.storage.googleapis.com/${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip -o /tmp/chromedriver.zip && \
+    export CHROME_VERSION=$(curl -s https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_STABLE) && \
+    curl -s "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}-1_amd64.deb" -o "/tmp/google-chrome-stable_${CHROME_VERSION}.deb" && \
+    (dpkg -i /tmp/google-chrome-stable_${CHROME_VERSION}.deb || true) && \
+    apt-get install -y -f && \
+    curl -s https://storage.googleapis.com/chrome-for-testing-public/${CHROME_VERSION}/linux64/chromedriver-linux64.zip -o /tmp/chromedriver.zip && \
```

Comment:
i would try to not hardcode a cpu arch, we would like to support arm in the near future

Reply:
unfortunately, CfT endpoints do not offer a `linux/arm64` option (just one for mac).
https://getwebdriver.com/chromedriver#stable

there is a compiled one in the ubuntu repositories for chromium + chromium-driver, both fixed at the same version. Under the assumption that they are updated at the same time, we could switch to installing chromium + chromium-driver. but I couldn't find out if superset supports chromium and testing is hardly possible without an environment.

So right now, I see two options:

Reply:
I'm going to test if chromium + chromium driver is compatible with superset.
(beforehand we used google chrome stable)
Will report back as soon as I have more information

Reply:
Hi! 
I tested these updates on our live environment, and we encountered no issues, indicating that Chromium and Chromium Driver are compatible with Apache Superset.

Reply:
@zyclonite can we release this? 🤩